src/search.js

```diff
@@ -51,8 +51,11 @@ export default class Search {
   }
 
   search() {
+    // return all docs if the must obj is empty
     const matches = this.filterResults(
-      this.searchFields(this.query.must),
+      Object.keys(this.query.must).length === 0 ?
```

Comment:
I don't think ES allows empty searches, and I feel like we should follow their docs as much as possible:
https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-all-query.html